tests/SprykerTest/Zed/EventBehavior/_support/_generated/EventBehaviorBusinessTesterActions.php

```diff
@@ -1,4 +1,4 @@
-<?php  //[STAMP] fc646900bab47048b53a79d3a36c8cf7
+<?php  //[STAMP] 6543653be52273f4c175eced1304a144
```

Comment:
Shouldnt this file be not commited in git??

Reply:
I have no idea. @ehsanmx ?

Reply:
@dereuromark yes you are right, these files are generated files and should not be committed into git

Reply:
You can make follow up fix PR (merge only)